CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add setting that allows switching between the project and user directories for the internal Sentry database location on Windows/Linux ([#616](https://github.com/getsentry/sentry-unreal/pull/616))
 - Add non-ASCII characters support for user messages ([#624](https://github.com/getsentry/sentry-unreal/pull/624))
+- Upgrade Android gradle plugin from v2.1.5 to v4.11.0 ([#633](https://github.com/getsentry/sentry-unreal/pull/633))
```

Comment:
@tustanivsky @bitsandfoxes would this require a major bump of this SDK?

Or since we're still experimental (`0.x`) at least a callout in the changelog larger than this one liner?

Reply:
This isn't a breaking change so a major version bump is not required. However it'll make sense to mention in docs that there's a fallback allowing users to switch back to the legacy gradle plugin in case any issues occur.